### ISSUE TITLE
Launch with FireCloud Should be Hidden or Disabled if workflows are invalid

### ISSUE THREAD

**Issue by coverbeck** [ga4gh/dockstore#1372]
## Bug Report

If viewing a version of a checker workflow that does not have the checker file, the Launch with FireCloud button is present.

### Expected behaviour

Button should be disabled or hidden

### Actual behaviour

It is clickable

### Steps to reproduce the behaviour 

1. Goto `https://dockstore.org/workflows/github.com/DataBiosphere/topmed-workflows/Functional_Equivalence_wdl_checker:1.1.01
2. See how Launch with FireCloud button is enabled, with the link `https://portal.firecloud.org/#import/dockstore/github.com/DataBiosphere/topmed-workflows/Functional_Equivalence_wdl_checker:1.1.0`
3. As proof that there is no workflow associated with the checker, click on the Files tab; you'll see the message `A Descriptor File associated with this Docker container could not be found.`.

Possibly related to #1371.

**Comment by denis-yuen** [comment]
Hmmm, 
1) https://dockstore.org/workflows/github.com/DataBiosphere/topmed-workflows/Functional_Equivalence_wdl_checker:1.1.01 isn't actually a version. It kicks me to the default which is checker/checker-fix Maybe https://dockstore.org/workflows/github.com/DataBiosphere/topmed-workflows/Functional_Equivalence_wdl_checker:1.1.0 was intended?
2)  I do see that version 1.1.0 doesn't have files, feature/checker-fix does. Looking at the versions page, only develop and feature/checker-fix are valid

TLDR: I think we need to limit launch-with to valid workflows

**Comment by garyluu** [comment]
The firecloud button is now hidden in develop.  Might be better disabled with a tooltip that says only valid descriptors with no secondary files work since it took me a while to figure out ones with secondary files are also hidden too

**Comment by denis-yuen** [comment]
seems functional as of
DockstoreAPI - 1.5.0-alpha.7
DockstoreUI - 2.2.0-alpha.5

on staging
